redux-devtools/redux-devtools.d.ts

```diff
@@ -14,24 +14,24 @@ declare module "redux-devtools" {
 declare module "redux-devtools/lib/react" {
```

Comment:
Again, I'm not using `redux-devtools`.  I just made these changes to the tests would pass.